src/main/java/core/basesyntax/service/ParseTransactionService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface ParseTransactionService {
+    List<FruitTransaction> parseStringTransactions(String transactions);
```

Comment:
did you use the last version of FruitShop here (approved version)? If not, update PR